source/jormungandr/jormungandr/interfaces/v1/JourneyCommon.py

```diff
@@ -0,0 +1,244 @@
+        parser_get.add_argument("last_section_mode[]",
+                                type=option_value(modes),
+                                dest="destination_mode", action="append")
+        # for retrocompatibility purpose, we duplicate (without []):
+        parser_get.add_argument("first_section_mode",
+                                type=option_value(modes), action="append")
+        parser_get.add_argument("last_section_mode",
+                                type=option_value(modes), action="append")
+        # no default value for data_freshness because we need to maintain retrocomp with disruption_active
+        parser_get.add_argument("data_freshness",
+                                type=option_value(['base_schedule', 'adapted_schedule', 'realtime']))
+        parser_get.add_argument("disruption_active", type=boolean, default=False)  # for retrocomp
+        parser_get.add_argument("to", type=unicode, dest="destination")
+        parser_get.add_argument("datetime", type=date_time_format)
+        parser_get.add_argument("max_duration", type=unsigned_integer)
+        parser_get.add_argument("datetime_represents", dest="clockwise",
+                                type=dt_represents, default=True)
+        parser_get.add_argument("forbidden_uris[]", type=unicode, action="append")
+        parser_get.add_argument("max_transfers", type=int, default=42)
+        parser_get.add_argument("_current_datetime", type=date_time_format, default=datetime.utcnow(),
+                                description="The datetime used to consider the state of the pt object"
+                                            " Default is the current date and it is used for debug."
+                                            " Note: it will mainly change the disruptions that concern "
+                                            "the object The timezone should be specified in the format,"
+                                            " else we consider it as UTC")
+        parser_get.add_argument("max_walking_duration_to_pt", type=int,
+                                description="maximal duration of walking on public transport in second")
+        parser_get.add_argument("max_bike_duration_to_pt", type=int,
+                                description="maximal duration of bike on public transport in second")
+        parser_get.add_argument("max_bss_duration_to_pt", type=int,
+                                description="maximal duration of bss on public transport in second")
+        parser_get.add_argument("max_car_duration_to_pt", type=int,
+                                description="maximal duration of car on public transport in second")
+        parser_get.add_argument("walking_speed", type=float_gt_0)
+        parser_get.add_argument("bike_speed", type=float_gt_0)
+        parser_get.add_argument("bss_speed", type=float_gt_0)
+        parser_get.add_argument("car_speed", type=float_gt_0)
+
+    def get(self, region=None, uri=None):
```

Comment:
this is not a `get` method, I think this is a `parse_args` method no ? (or other name)